packages/suite-native-atoms/src/Input/Input.tsx

```diff
@@ -14,20 +21,26 @@ type InputProps = {
     leftIcon?: ReactNode;
 };
 
+const INPUT_WRAPPER_PADDING_HORIZONTAL = 14;
+const INPUT_WRAPPER_PADDING_VERTICAL = 17;
+const INPUT_WRAPPER_PADDING_VERTICAL_SHRANK = nativeSpacings.small;
+const INPUT_TEXT_HEIGHT = 24;
+
 type InputWrapperStyleProps = {
     hasWarning: boolean;
     hasError: boolean;
+    isLabelShrank: boolean;
```

Comment:
I probably don't like this name, it was kind of cryptic for first sight for me. Any other options? I would maybe at least prefer `isLabelShrink`, because there is no reason for past tense, it's shrinked right now :D

Reply:
I would maybe go for `isLabelTopPositioned` or something like that?

Reply:
`react-native-paper` uses `isLabelMinized` which I kind of like

https://github.com/callstack/react-native-paper/blob/main/src/components/TextInput/TextInputFlat.tsx

Reply:
I agree that `isLabelShrank` is not so clear. Maybe what I like the most is `isLabelMinimized`. I cannot even find `isLabelMinized` in react-native-paper codebase.

Reply:
they use just `minimized` keyword for this stuff, not exactly our name